### ISSUE TITLE
AttributeError: 'SQLAlchemyAdapter' object has no attribute 'config'

### ISSUE THREAD

**Issue by tuwid** [lingthio/Flask-User#158]
Hi,
I just pasted the code from the example in your documentation and i get this error 
```
vagrant-ubuntu-trusty-64:/vagrant/test# python basic_app.py
Traceback (most recent call last):
  File "basic_app.py", line 86, in <module>
    app = create_app()
  File "basic_app.py", line 52, in create_app
    user_manager = UserManager(app, db_adapter)     # Initialize Flask-User
  File "/usr/local/lib/python2.7/dist-packages/flask_user/__init__.py", line 53, in __init__
    self.init_app(app, db_adapter, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/flask_user/__init__.py", line 146, in init_app
    settings.set_default_settings(self, app.config)
AttributeError: 'SQLAlchemyAdapter' object has no attribute 'config'
```
I'm using Python 2.7.6 (default, Oct 26 2016, 20:30:19)
pip 9.0.1 from /usr/local/lib/python2.7/dist-packages (python 2.7)

**Comment by lingthio** [comment]
Hi @tuwid , thank you for reporting this problem.

I managed to upload the v0.9 docs into the v0.6 documentation. My bad. I corrected the v0.6 docs on pythonhosted.

You need to reverse the parameter order like so::

    user_manager = UserManager(db_adapter, app)

**Comment by tuwid** [comment]
apparently this works, thanks
